policykit/policyengine/admin.py

```diff
@@ -156,7 +158,8 @@ def save_model(self, request, obj, form, change):
 
 
 class ConstitutionPolicyBundleAdmin(admin.ModelAdmin):
-    fields= ('bundled_policies', 'description')
+    fields= ('bundled_policies',)
+    
```

Comment:
why are we removing description? i may want to add a description to my bundled policy (also a name)?

Reply:
I removed description because models.py did not specify a description field (only a policy_type which was not a model field) so I had assumed that the field description was maybe an error. Should I add both bundle policy name and description as model fields?

Reply:
But it's a subclass of a class that does have a description, right? (BasePolicy)?